ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -447,7 +447,10 @@ objects:
           done < /tmp/containers
 
           echo "Gathering kube-apiserver audit.log ..."
-          FILTER=gzip queue /tmp/artifacts/nodes/masters-kube-apiserver-audit.log.gz oc adm node-logs --role=master --path=kube-apiserver/audit.log
+          FILTER=gzip queue /tmp/artifacts/nodes/masters-kube-apiserver-audit.log.gz oc adm node-logs --role=master --path=kube-apiserver/
```

Comment:
This is just going to show you a directory listing unless you changed something in Kubelet...